scene/main/scene_tree.cpp

```diff
@@ -596,7 +611,9 @@ void SceneTree::process_timers(double p_delta, bool p_physics_frame) {
 		E->get()->set_time_left(time_left);
 
 		if (time_left <= 0) {
-			E->get()->emit_signal(SNAME("timeout"));
+            if (E->get()->get_owner() && E->get()->get_owner()->is_inside_tree() ) {
+                E->get()->emit_signal("timeout");
```

Comment:
```suggestion
                E->get()->emit_signal(SNAME("timeout"));
```
No reason to remove this